src/collection.rs

```diff
@@ -12,13 +12,24 @@ use std::io::prelude::*;
 use std::path::PathBuf;
 
 #[derive(Clone, Debug, Deserialize, PartialEq)]
+#[serde(rename_all(serialize = "PascalCase"))]
 pub struct Collection {
+    #[serde(alias = "Version")]
     pub version: String,
-    #[serde(default)]
+    #[serde(default, skip_serializing_if = "Option::is_none", alias = "Origin")]
     pub origin: Option<String>,
-    #[serde(rename = "component", default)]
+    #[serde(
+        rename(deserialize = "component", serialize = "Component"),
```

Comment:
Do we really need to specify the serialize part?

Reply:
Yes. In XML it's `component`, and for JSON it's `Component`. Otherwise the upper/lower case would not be correct.

Reply:
Right but alias should fix it already

Reply:
Yup you're right, my bad. Fixed it.

Reply:
Aaah nope. It is required for serializing :(
Serializing doesn't care about the alias, so it takes the lowercase name.